watertap/flowsheets/electrodialysis/electrodialysis_1stack_conc_recirc_ui.py

```diff
@@ -240,23 +240,23 @@ def export_variables(flowsheet=None, exports=None, build_options=None, **kwargs)
     )
```

Comment:
Do you need the add the `spacer_conductivity_coefficient` as a UI variable?

Reply:
no and not all variables are subject to UI.